readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
```

Comment:
I assume this spec will be used only on the version 2, if there is another version, we will need to create another schema.

Also, I was thinking that we can use this to validate the current v1 API. But I'm not sure if that's worth it.

Reply:
Yea, I think we can  keep v1 as it is.